students/tonylee/SecondAssignment.py

```diff
@@ -0,0 +1,69 @@
+def exchange_first_last(seq):
+    	s = list(seq)
+	first = seq[0]
+	last = seq[-1]
+	s[0] = last
+	s[-1] = first
+	
+	return convert_format(seq, s)
+
+def remove_every_other_item(seq):
+	s = list(seq)
+        l = []
+	i = 0
+	while (i < len(s)):
+		if i%2 != 0:
+			l.append(s[i])
+		i = i + 1
+	
+	return convert_format(seq, l)
+
+def remove_first4_last4_every_other_item(seq):
+	s = list(seq)
+	del s[0:4]
+	del s[len(s)-4:len(s)]
+	
+	return convert_format(seq, s)
+	
+def reverse_element(seq):
+	s = list(seq)
+	r = s[::-1]
+	
+	return convert_format(r)
```

Comment:
This function doesn't seem to be worked correctly.  Getting TypeError: convert_format() missing 1 required positional argument: 'value'